src/wp-includes/post.php

```diff
@@ -3418,25 +3417,22 @@ function wp_count_posts( $type = 'post', $perm = '' ) {
 				FROM {$wpdb->posts}
 				WHERE post_type = %s AND post_status = 'private' AND post_author = %d
 			) AS filtered_posts
-			",
-			$type,
-			$type,
-			get_current_user_id()
-		);
+		";
+		$query_params = array( $type, $type, get_current_user_id() );
```

Comment:
I'm renaming this to `$query_args` since `$args` is the variable name used in `$wpdb->prepare()`.

Reply:
Actually, in 9cb8ea342a I'm renaming to just `$args` in this case